entrypoint.sh

```diff
@@ -9,4 +9,4 @@ mkdir staticfiles
 python /api/manage.py collectstatic
 
 # Set tmp dir to be in-memory for speed. Pass logs to stdout/err as Docker will expect them there
-gunicorn --worker-tmp-dir /dev/shm --timeout 600 --log-file=- --bind :8000 --workers 3 api.wsgi:application
+gunicorn --worker-tmp-dir /dev/shm --timeout 600 --log-file=- --bind :8000 --workers 3 api.wsgi:application --reload
```

Comment:
oh yeh @spco I had to app this to get the app to detect changes I have made to files in api.py, not sure why this became a problem for me, or if others had had this problem.

https://stackoverflow.com/questions/12773763/gunicorn-autoreload-on-source-change

Reply:
You're probably right, and there's hopefully no harm in it in the deployed setting either.